tests/integration-tests/test_runner.py

```diff
@@ -705,7 +730,11 @@ def _check_args(args):
         assert_that(args.schedulers).described_as("--schedulers cannot be empty").is_not_empty()
     else:
         try:
-            args.tests_config = _test_config_file(args.tests_config)
+            test_config_args = get_config_arguments(args)
+            if test_config_args:
+                args.tests_config = _test_config_file(args.tests_config, test_config_args)
```

Comment:
same here, you can always pass ahead the test_config_args even if it's None, because the method is able to understand when to use it.